cips/cip-26.md

```diff
@@ -0,0 +1,64 @@
+| cip | 26 |
+| - | - |
+| title | Reduce block time to six seconds |
+| description | Proposal to reduce block time on celestia-app to 6 seconds, from 12. |
+| author | Josh Stein ([@jcstein](https://github.com/jcstein)), Rootul Patel ([@rootulp](https://github.com/rootulp))|
+| discussions-to | <https://forum.celestia.org/t/cip-decrease-block-time-to-6-seconds/1836> |
+| status | Draft |
+| type | Standards Track |
+| category | Core |
+| created | 2024-10-09 |
+
+## Abstract
+
+This CIP proposes to reduce the block time on celestia-app to 6 seconds, from 12. This will double the throughput and reduce the time it takes for transactions to be finalized in half. This CIP also proposes to increase the `ttl-num-blocks` parameter in the mempool configuration from 5 to 12 to maintain consistency with the new block time.
```

Comment:
>This will double the throughput and reduce the time it takes for transactions to be finalized in half

This is not entirely necessarily correct, in the sense that the users wont necessarily see 2x bytes per second after this change, for example, blocks can be empty or not full, hence the throughput would not be doubled. Also the same for finalization of txs, it depends on other factors, and while reducing block time has an impact, it does not exactly cut the finalization in half.

Reply:
Assuming blocks are being filled, this would double throughput (2MB/12s -> 2MB/6s) and cut down finalization from 12s to 6s? right? what would you suggest this changes to?

Reply:
>Assuming blocks are being filled

That is exactly my point, throughput is a function of block size as well, not just block time, as you also mentioned in your comment "Assuming blocks are being filled".

Reply:
> cut down finalization from 12s to 6s?

In addition to the block time, txs priority also contribute to their finality. So, it is a multifold problem, hence cutting block time does not necessarily make all txs to be finalized faster, in my pov.